assets/onboarding/features/index.jsx

```diff
@@ -0,0 +1,207 @@
+		),
+		learnMoreLink:
+			'https://woocommerce.com/products/sensei-course-progress/',
+		status: SUCCESS_STATUS,
+	},
+	{
+		id: 'certificates',
+		title: __( 'Certificates', 'sensei-lms' ),
+		description: __(
+			'Award your students with a certificate of completion and a sense of accomplishment after finishing a course.',
+			'sensei-lms'
+		),
+		learnMoreLink: 'https://woocommerce.com/products/sensei-certificates/',
+		errorMessage: __(
+			'Error message here, maecenas faucibus mollis interdum tristique euismod.',
+			'sensei-lms'
+		),
+		status: ERROR_STATUS,
+	},
+	{
+		id: 'media_attachments',
+		title: __( 'Media Attachments', 'sensei-lms' ),
+		description: __(
+			'Provide your students with easy access to additional learning materials, from audio files to slideshows and PDFs.',
+			'sensei-lms'
+		),
+		learnMoreLink:
+			'https://woocommerce.com/products/sensei-media-attachments/',
+		status: SUCCESS_STATUS,
+	},
+	{
+		id: 'content_drip',
+		title: __( 'Content Drip', 'sensei-lms' ),
+		price: '$29.00 per year',
+		description: __(
+			'Keep students engaged and improve knowledge retention by setting a delivery schedule for course content.',
+			'sensei-lms'
+		),
+		learnMoreLink: 'https://woocommerce.com/products/sensei-content-drip/',
+		status: LOADING_STATUS,
+	},
+].map( ( feature ) => ( {
+	...feature,
+	title: `${ feature.title } — ${
+		feature.price ? feature.price : __( 'Free', 'sensei-lms' )
+	}`,
+} ) );
+
+/**
+ * Features step for setup wizard.
+ */
+const Features = () => {
+	const [ confirmationModalActive, toggleConfirmationModal ] = useState(
+		false
+	);
+	const [ showInstallationFeedback, toggleInstallationFeedback ] = useState(
+		false
+	);
+	const [ selectedFeatureIds, setSelectedFeatureIds ] = useState( [] );
+	const { goTo } = useQueryStringRouter();
+
+	const finishSelection = () => {
+		if ( 0 === selectedFeatureIds.length ) {
+			goToNextStep();
+			return;
+		}
+
+		toggleConfirmationModal( true );
+	};
+
+	const goToInstallation = () => {
+		toggleConfirmationModal( false );
+		toggleInstallationFeedback( true );
+	};
+
+	const goBackToSelection = () => {
+		toggleInstallationFeedback( false );
+	};
+
+	const goToNextStep = () => {
+		goTo( 'ready' );
+	};
+
+	const toggleItem = ( id ) => {
+		setSelectedFeatureIds( ( selected ) => [
+			...( selected.includes( id )
+				? selected.filter( ( item ) => item !== id )
+				: [ id, ...selected ] ),
+		] );
+	};
+
+	const getSelectedFeatures = () =>
+		features.filter( ( feature ) =>
+			selectedFeatureIds.includes( feature.id )
+		);
+
+	return (
+		<>
+			<div className="sensei-onboarding__title">
+				<H>
+					{ __(
+						'Enhance your online courses with these optional features!',
+						'sensei-lms'
+					) }
+				</H>
+			</div>
+			<Card className="sensei-onboarding__card">
+				{ showInstallationFeedback ? (
+					<InstallationFeedback
+						features={ getSelectedFeatures() }
+						onContinue={ goToNextStep }
+					/>
+				) : (
+					<>
```

Comment:
I wonder if it would make sense to pull this section into its own component as well? That would simplify this piece of the render, I think?

Reply:
Yeah! It looks better now: https://github.com/Automattic/sensei/pull/3121/commits/bfafba20d2f9494ae8de34f9824aa2699347f038